mappings/net/minecraft/item/ItemGroup.mapping

```diff
@@ -75,23 +78,32 @@ CLASS net/minecraft/unmapped/C_lnokcayd net/minecraft/item/ItemGroup
 	CLASS C_lbxcoexd EntrySupplier
 		METHOD accept accept (Lnet/minecraft/unmapped/C_czxxrbcp;Lnet/minecraft/unmapped/C_lnokcayd$C_wnogvxsu;Z)V
 			ARG 1 enabledFeatures
+			ARG 2 collector
+			ARG 3 op
 	CLASS C_ouweheor Type
 	CLASS C_qjyuivjw GatedStackCollector
 		FIELD f_alfldgtl enabledFeatures Lnet/minecraft/unmapped/C_czxxrbcp;
 		FIELD f_ewkqlato searchStacks Ljava/util/Set;
 		FIELD f_gynphoyu tabStacks Ljava/util/Collection;
 		METHOD <init> (Lnet/minecraft/unmapped/C_lnokcayd;Lnet/minecraft/unmapped/C_czxxrbcp;)V
 			ARG 2 enabledFeatures
-	CLASS C_qwchbyug Location
+	CLASS C_qwchbyug VerticalPosition
```

Comment:
could you provide a bit of a description on what this is?

Reply:
It's the position where the tab will be vertically; If it's TOP, it's on the inventory's top, if it's BOTTOM, it's in the bottom; oh, this is an enum by the way

Reply:
in that case I like `VerticalLocation`, although `position` is good too!
personally not a fan of `orientation`
I'd also add a short javadoc comment explaining that because you can't have too many docs

Reply:
Hm, I'll think about it